library/src/main/java/io/constructor/data/DataManager.kt

```diff
@@ -49,6 +49,14 @@ constructor(private val constructorApi: ConstructorApi, private val moshi: Moshi
         }.toObservable()
     }
 
+    suspend fun getAutocompleteResultsCRT(term: String, encodedParams: Array<Pair<String, String>> = arrayOf()): AutocompleteResponse {
+        var dynamicUrl = "/${ApiPaths.URL_AUTOCOMPLETE.format(term)}"
+        encodedParams.forEachIndexed { index, pair ->
+            dynamicUrl += "${if (index != 0) "&" else "?" }${pair.first}=${pair.second}"
+        }
```

Comment:
Seems like this can be extracted to a function since it's reused just with different `dynamicURL`s.

Reply:
Love it. Made some updates! 😄